internal/cloud/realm/secrets.go

```diff
@@ -0,0 +1,42 @@
+package realm
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/10gen/realm-cli/internal/utils/api"
+)
+
+const (
+	valuesPathPattern = appPathPattern + "/values"
+)
+
+// Value is a value or secret stored in a Realm application
+type Value struct {
+	ID           string `json:"_id"`
+	Name         string `json:"name"`
+	LastModified int64  `json:"last_modified"`
+	Secret       bool   `json:"from_secret"`
+}
+
+func (c *client) FindValues(app App) ([]Value, error) {
+	res, resErr := c.do(
+		http.MethodGet,
+		fmt.Sprintf(valuesPathPattern, app.GroupID, app.ID),
+		api.RequestOptions{},
+	)
+	if resErr != nil {
+		return nil, resErr
+	}
+
```

Comment:
with the recent refactor, we actually have a standardized approach to this scenario now:

```go
res, resErr := c.do(...)
if resErr != nil {
  return nil, resErr
}
if res.StatusCode != http.StatusOK {
  return nil, api.ErrUnexpectedStatusCode{"get values", res.StatusCode}
}
defer res.Body.Close()
```

Reply:
Sounds good, I can rebase once the push PR goes in. I imagine there will other standardizations to integrate as well